go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -624,15 +627,15 @@ func (be *BuiltinBackupEngine) restoreFile(ctx context.Context, cnf *Mycnf, bh b
 	if compress {
 		gz, err := pgzip.NewReader(reader)
 		if err != nil {
-			return err
+			return fmt.Errorf("can't open gzip decompressor: %v", err)
 		}
 		defer func() {
 			if cerr := gz.Close(); cerr != nil {
 				if err != nil {
 					// We already have an error, just log this one.
 					log.Errorf("failed to close gunziper %v: %v", name, cerr)
```

Comment:
can you change this log message to match the error in the else part?

Reply:
Done.